src/traces/waterfall/attributes.js

```diff
@@ -0,0 +1,113 @@
+/**
+* Copyright 2012-2019, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var barAttrs = require('../bar/attributes');
+var lineAttrs = require('../scatter/attributes').line;
+var extendFlat = require('../../lib/extend').extendFlat;
+
+function directionAttrs() {
+    return {
+        editType: 'style',
+        color: barAttrs.marker.color,
```

Comment:
We should override the description here and for `marker.line.color` as waterfall trace don't support numeric colorscales.

Reply:
Moreover, what currently happens when we have

```js
{
  type: 'waterfall',
  x: [/* */],
  y: [/* */],
  marker: {color: [1,2,3]}
}
```

?

Reply:
Now they turn to black. So you are right about overriding the description.

Reply:
> Now they turn to black. 

... which I guess is ok? Are we planning on ever adding colorscale support for waterfall traces?

Reply:
I think they could be pretty useful for cases to illustrate `extreme` jumps. 
Here is one example found on the web: https://i.ytimg.com/vi/tapO6Dcf9i4/maxresdefault.jpg
cc: @nicolaskruchten @alexcjohnson

Reply:
... We still need to override the description.